src/extension.test.ts

```diff
@@ -30,3 +30,39 @@ suite('Extension Intialization Tests', () => {
 		assert.equal(imljsonLanguageClient?.isRunning(), true);
 	});
 });
+
+/**
+ * See https://stackoverflow.com/questions/38279920/how-to-open-file-and-insert-text-using-the-vscode-api
+ * how to open file
+ */
+suite('Language ID tests for files used by online edits', () => {
+	const filenamesForOnlineEdit = [
+		{ filename: 'parameters.imljson', language: 'imljson' },
+		{ filename: 'expect.imljson', language: 'imljson' },
+		{ filename: 'interface.imljson', language: 'imljson' },
+		{ filename: 'common.imljson', language: 'imljson' },
```

Comment:
is common imljson type?

Reply:
can you extend also to `readme`?
Or it is processed differently?

Reply:
README is the standard markdown. There is nothing added in our extension for processing this file type.